buildSrc/src/main/java/dependencies/Versions.kt

```diff
@@ -3,4 +3,11 @@ package dependencies
 private object Versions {
     val androidCompileSdkVersion = 28
     val androidMinSdkVersion = 21
+
+    private val versionMajor = 1
+    private val versionMinor = 0
+    private val versionPatch = 0
+    val androidvVersionCode = versionMajor * 10000 + versionMinor * 100 + versionPatch
```

Comment:
it looks like maybe a typo 👀 should it be `androidVersionCode` instead?

Reply:
OMG, Sorry!
I fixed it!